src/Okta.Sdk.IntegrationTests/PoliciesScenarios.cs

```diff
@@ -0,0 +1,1482 @@
+﻿// <copyright file="PoliciesScenarios.cs" company="Okta, Inc">
+// Copyright (c) 2014 - present Okta, Inc. All rights reserved.
+// Licensed under the Apache 2.0 license. See the LICENSE file in the project root for full license information.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Xunit;
+
+namespace Okta.Sdk.IntegrationTests
+{
+    [Collection(nameof(PoliciesScenarios))]
+    public class PoliciesScenarios
+    {
+        [Fact]
+        public async Task CreateSignOnPolicy()
+        {
+            var client = TestClient.Create();
+
+            IPolicy policy = new Policy()
+            {
+                // Name has a maximum of 50 chars
+                Name = $"Default policy {Guid.NewGuid()}".Substring(0, 50),
```

Comment:
@vijetmahabaleshwar-okta do you want us to put something in here that indicates that this test is being done by this SDK?